lualib/resty/events/compat/init.lua

```diff
@@ -14,7 +16,7 @@ local handlers = {}
 local _configured
 
 local _M = {
-    _VERSION = '0.1.0',
+    _VERSION = events._VERSION,
```

Comment:
Perhaps we should keep it as `0.1.0`, lua-resty-healthcheck may check this version number.

Reply:
Seems the current version in the rest.events.init.lua is `0.2.0`. Would this be an inconsistency?

https://github.com/Kong/lua-resty-events/blob/7906f0c28e8081622bbb34d47da69bcf8abe944d/lualib/resty/events/init.lua#L16

Reply:
Currently, outside does not use `resty.events` directly, `resty.events.compat` is the real API in fact.

Reply:
So, the result is: resty.events.compat as 0.1.0 and other components (the new lib) uses 0.2.0.

Reply:
Or, returned it back to `0.1.0`.